client/blocks/reader-full-post/index.jsx

```diff
@@ -551,7 +551,11 @@ export class FullPostView extends React.Component {
 								/>
 							) }
 
-							<div className="reader-full-post__comments-wrapper" ref={ this.commentsWrapper }>
+							<div
+								id="comments"
+								className="reader-full-post__comments-wrapper"
+								ref={ this.commentsWrapper }
```

Comment:
It looks like this comments wrapper is used to scroll it into focus after a click on the comments link. It may be good to move focus to this element as well. At the moment it resets focus to the top of the page since it's a new page (expected behavior), but when someone clicks a comment(s) link, they would expect to be placed in a comments section. This could be a follow-up PR, as it may require a few more changes. Something like:

- Change from `div` to `section` to give it a tiny semantics boost
- Add a hidden "Comments" heading (`<h2>`? `<h3>`? Not sure which one) with an `aria-labelledby` referencing the `id` from the heading. This along with the `section` should get it to show-up in the landmarks menu. If not, then the heading is really useful anyways.
- Add a `tabindex="-1"` so it's programmatically focusable.
- Focus the element when it's available. Can likely place this somewhere around the `scrollTo` that's used.

Reply:
I tested this with a screen reader, and focus is totally lost on the page change. It's like it's stuck "behind" on the old feed or something. This may be a deeper issue of always needing to move focus to the full page view after a click. We can at least fix this one though, and probably worth follow-up.